Event.php

```diff
@@ -73,12 +79,6 @@ class Event extends Component
      * @var string
      */
     protected $_description;
-    /**
-     * The mutex implementation.
-     *
-     * @var \yii\mutex\Mutex
-     */
-    protected $_mutex;
```

Comment:
Why did you remove mutex? it is used for withoutOverlapping

Reply:
I am sorry the code should not be deleted.

Reply:
Thanks,I will start a new pull request soon.